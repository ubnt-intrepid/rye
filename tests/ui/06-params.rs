fn main() {}

#[rye::test(rye_path = 10)]
fn invalid_path_type() {}

#[rye::test(rye_path = "catcher_in_the_rye")]
fn unresolved_path() {}

#[rye::test(rye_path = "rye", rye_path = "rye")]
fn duplicated() {}

#[rye::test(the_quick_fox = "lazy")]
fn unknown_param() {}
