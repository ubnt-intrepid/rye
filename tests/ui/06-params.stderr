error: required a string literal
 --> $DIR/06-params.rs:3:24
  |
3 | #[rye::test(rye_path = 10)]
  |                        ^^

error: duplicated parameter
 --> $DIR/06-params.rs:9:31
  |
9 | #[rye::test(rye_path = "rye", rye_path = "rye")]
  |                               ^^^^^^^^^^^^^^^^

error: unknown parameter name
  --> $DIR/06-params.rs:12:13
   |
12 | #[rye::test(the_quick_fox = "lazy")]
   |             ^^^^^^^^^^^^^

error[E0433]: failed to resolve: use of undeclared type or module `catcher_in_the_rye`
 --> $DIR/06-params.rs:6:24
  |
6 | #[rye::test(rye_path = "catcher_in_the_rye")]
  |                        ^^^^^^^^^^^^^^^^^^^^ use of undeclared type or module `catcher_in_the_rye`
